src/ctap/ctap1.rs

```diff
@@ -292,22 +255,22 @@ impl Ctap1Command {
         let pk = sk.genpk();
         let key_handle = ctap_state
             .encrypt_key_handle(sk, &application)
-            .map_err(|_| Ctap1StatusCode::SW_COMMAND_ABORTED)?;
+            .map_err(|_| Ctap1StatusCode::SW_INTERNAL_EXCEPTION)?;
         if key_handle.len() > 0xFF {
             // This is just being defensive with unreachable code.
-            return Err(Ctap1StatusCode::SW_VENDOR_KEY_HANDLE_TOO_LONG);
+            return Err(Ctap1StatusCode::SW_WRONG_LENGTH);
```

Comment:
Please don't change the return codes as this may break interop.

Reply:
In this case, the if should be unreachable and we're just being defensive here. I think this should be `SW_INTERNAL_EXCEPTION` with our new policy.
The risk of interoperability issues is low in this particular case, but I see the point of interop breaking changes. So I'm just leaving my opinion here, if we want to get rid of `SW_VENDOR_KEY_HANDLE_TOO_LONG`.

Reply:
updated to `SW_INTERNAL_EXCEPTION` as per team chat